### PR TITLE
Run grunt

### PR DESCRIPTION
**Pull request by Blendify**
- Update grunt file with changes done to docs
- Update grunt sass to 1.0.0 to avoid warning message

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -133,17 +133,17 @@ module.exports = function(grunt) {
       },
       /* Changes in theme dir rebuild sphinx */
       sphinx: {
-        files: ['sphinx_rtd_theme/**/*', 'demo_docs/**/*.rst', 'demo_docs/**/*.py'],
+        files: ['sphinx_rtd_theme/**/*', 'docs/**/*.rst', 'docs/**/*.py'],
```
Comment (jessetan) [inline Gruntfile.js]: Since README is now also built, `README.rst` should be added to wacthed files.

Comment (Blendify) [reply Gruntfile.js]: Can we just check the full docs file? `docs/**/*` wont this catch everything?

Comment (jessetan) [reply Gruntfile.js]: README.rst is not in the `docs` dir, it's in the root.

File: sphinx_rtd_theme/static/js/theme.js [hunk 2]
```diff
@@ -16,17 +16,22 @@ function ThemeNav () {
         isRunning: false
     };
 
-    nav.enable = function () {
+    nav.enable = function (withStickyNav) {
```
Comment (agjohnson) [inline sphinx_rtd_theme/static/js/theme.js]: What is the purpose of this addition? If configurable, this needs more information or documentation.

File: sphinx_rtd_theme/static/js/theme.js [hunk 3]
```diff
@@ -80,9 +90,16 @@ function ThemeNav () {
...
-        $("table.docutils:not(.field-list)")
+        $("table.docutils:not(.field-list,.footnote,.citation)")
             .wrap("<div class='wy-table-responsive'></div>");
 
+        // Add extra class to responsive tables that contain
+        // footnotes or citations so that we can target them for styling
+        $("table.docutils.footnote")
+            .wrap("<div class='wy-table-responsive footnote'></div>");
+        $("table.docutils.citation")
+            .wrap("<div class='wy-table-responsive citation'></div>");
```
Comment (agjohnson) [inline sphinx_rtd_theme/static/js/theme.js]: These changes are also unrelated to the PR. Is there background on these changes anywhere?

This pattern always seemed strange, especially when we have full control of the Sphinx writer as a Sphinx extension. The most correct is to do this on writing, not on each view I think. It might be best to split this work out into a separate PR if it isn't already.

File: sphinx_rtd_theme/static/js/theme.js [hunk 4]
```diff
@@ -170,7 +187,7 @@ function ThemeNav () {
 module.exports.ThemeNav = ThemeNav();
 
 if (typeof(window) != 'undefined') {
-    window.SphinxRtdTheme = { StickyNav: module.exports.ThemeNav };
+    window.SphinxRtdTheme = { Navigation: module.exports.ThemeNav };
```
Comment (agjohnson) [inline sphinx_rtd_theme/static/js/theme.js]: This likely breaks on RTD. Has this been tested?

**Comment by Blendify** [comment]
All the changes done to `theme.js` were made in separate PRs run git blame to see the changes.

File: sphinx_rtd_theme/static/js/theme.js (hunk 2 above)
Comment (jessetan) [reply sphinx_rtd_theme/static/js/theme.js]: This comes from #519. `enable()` was only called to enable sticky nav, while it was needed to decorate the navbar with the correct toggle options.
The extra bool was added so that a single entry point could be used to start the navigation logic (with or without sticky nav).
Documenting (with JSDoc?) of this file is a good idea, perhaps something for a separate issue?

File: sphinx_rtd_theme/static/js/theme.js (hunk 3 above)
Comment (jessetan) [reply sphinx_rtd_theme/static/js/theme.js]: This comes from #488, see comments there.

File: sphinx_rtd_theme/static/js/theme.js (hunk 4 above)
Comment (jessetan) [reply sphinx_rtd_theme/static/js/theme.js]: This also comes from #519. The theme code that calls this has been [changed accordingly](https://github.com/rtfd/sphinx_rtd_theme/pull/519/files#diff-1336f80431f476a232e3c48abb277c51).
It was tested on a local build, but I don't know if it has been tested on RTD. I can't imagine this function being called from outside of the theme though. A [search in the RTD source](https://github.com/rtfd/readthedocs.org/search?utf8=%E2%9C%93&q=StickyNav&type=) does not turn up anything.

**Comment by jessetan** [comment]
This is a good example of confusion caused by the built JS being included in PRs, as discussed in #379

File: sphinx_rtd_theme/static/js/theme.js (hunk 2 above)
Comment (agjohnson) [reply sphinx_rtd_theme/static/js/theme.js]: Yeah, not important here I think, but having our public API documented would make sense for folks extending or repurposing the theme.

File: sphinx_rtd_theme/static/js/theme.js (hunk 3 above)
Comment (agjohnson) [reply sphinx_rtd_theme/static/js/theme.js]: I've opened #530 to experiment with removing this pattern.

File: sphinx_rtd_theme/static/js/theme.js (hunk 4 above)
Comment (agjohnson) [reply sphinx_rtd_theme/static/js/theme.js]: At some point we were overriding or referencing `StickyNav` in RTD. So this would be worth testing.

**Comment by agjohnson** [comment]
Agreed. Separating asset generation from feature PRs is a great help. In the interim, until we get more serious about our release process, separating asset generation into a changeset and noting the changeset in PR would make review slightly easier. We do something similar with linting review on RTD, but it's still difficult to review.

Changes here make more sense now, think this can just be merged.
